blueprints/initializer-test/mocha-files/__root__/__testType__/__path__/__name__-test.js

```diff
@@ -2,7 +2,7 @@ import { expect } from 'chai';
 import { describe, it, beforeEach, afterEach } from 'mocha';
 import { run } from '@ember/runloop';
 import Application from '@ember/application';
-import { initialize } from '<%= dasherizedModulePrefix %>/initializers/<%= dasherizedModuleName %>';
+import { initialize } from '<%= dasherizedModulePrefix %>/<% if (isModuleUnificationProject) { %>init/<% } %>initializers/<%= dasherizedModuleName %>';
```

Comment:
perhaps it would be simpler to replace `dasherizedModulePrefix` with `modulePrefix` in `locals` so that we don't have to litter lots of files with `<% if (isModuleUnificationProject) { %>init/<% } %>` statements?

Reply:
:+1: great suggestion!